.github/workflows/e2e-suite.yml

```diff
@@ -111,7 +116,14 @@ jobs:
         run: |
           timestamp=$(date +'%Y%m%d%H%M')
           report_filename="${timestamp}_cli_test_report.xml"
-          make testint TEST_ARGS="--junitxml=${report_filename}" MODULE="${{ inputs.module }}"
+          echo "Running tests, report: ${report_filename}"
+          if [ -n "$OPENAPI_SPEC_URL" ]; then
+            echo "Running tests with OpenAPI spec from: $OPENAPI_SPEC_URL"
+            make testint TEST_ARGS="--junitxml=${report_filename}" MODULE="${{ inputs.module }}"
+          else
+                echo "Running tests with default OpenAPI spec"
+                make testint TEST_ARGS="--junitxml=${report_filename}"
```

Comment:
I think you forgot the MODULE here. I think it would be nice if we just keep it simple and not echo anything in this step. 
e.g. `make testint TEST_ARGS="--junitxml=${report_filename}" MODULE="${{ inputs.module }}"`

Can you confirm one more time with forked run as well after the changes?

Reply:
![Screenshot 2024-11-20 at 9 56 39 PM](https://github.com/user-attachments/assets/c9a28540-266a-4fe5-ab5a-c780ac2c6703)
![Uploading Screenshot 2024-11-20 at 10.05.04 PM.jpg…]()